_components/input-mask/guidance/implementation.md

```diff
@@ -1,24 +1,84 @@
+- **Add a `mask` attribute to the input element.** The mask attribute is used to define the format of your mask. Use `9`s for numbers and `A`s for letters. For example, a typical SSN number format in the mask attribute should look like `999-99-9999`.
+- **Add a `placeholder` attribute to the input element.** This indicates the mask format. Use underscores `_` for character input placeholders combined with any valid format characters (see below) to customize your input mask. e.g. A US telephone `placeholder` attribute may look like `(___) ___-____`
+- **Add a `maxlength` attribute to the input element.** This will be used as the limit referenced in the message and for validation.
+- **Set the attribute `type` to numeric if your mask is all numbers.** This will display the numeric keypad instead of the full keyboard.
+
+{:.usa-content-list }
+The following is an example of `input` element attributes for a Social Security Number mask:
 
 {:.usa-content-list }
 ```
-placeholder="___ ___ ___"
-pattern="\d\d\d \d\d\d \d\d\d"
+aria-describedby="ssn-hint ssn-hint-info"
+class="usa-input usa-input-mask__field"
+id="ssn"
 inputmode="numeric"
+mask="999-99-9999"
+maxlength="11"
+name="ssn"
+placeholder="___-__-____"
+type="text"
 ```
 
 {:.usa-content-list }
 <h3>The input mask and alphanumeric characters</h3>
 
 {:.usa-content-list }
-Input mask examples include an alphanumeric version which can be customized to meet your needs. To implement an alphanumeric input mask, you will need to add the attribute `data-charset` to the input tag. Similar to the `pattern` attribute, include a number sign `#` for each number and an `A` for each required letter. We recommend changing inputmode to `text` so the full keyboard is available to your user.
+Input mask examples include an alphanumeric version which can be customized to meet your needs. Include the number `9` for every number and a capitalized letter `A` for every required letter. We recommend changing inputmode to `text` so the full keyboard is available to your user.
 
 {:.usa-content-list }
-If your input mask calls for a pattern such as _1A2 B3C_, you would implement that by adding the following attributes and character formatting to your input element:
+If your input mask calls for a complicated pattern like _1EG4-TE5-MK73_, you could implement that by adding the following attributes and character formatting to your `input` element:
 
 {:.usa-content-list }
 ```
-placeholder="___ ___"
-pattern="\w\d\w \d\w\d"
-data-charset="A#A #A#"
+aria-describedby="alphanumeric-hint alphanumeric-hint-info"
+class="usa-input usa-input-mask__field"
+forceupper="true"
+id="alphanumeric"
+mask="9AA9-AA9-AA99"
+name="alphanumeric"
+placeholder="____-___-____"
+maxlength="13"
+type="text"
```

Comment:
This code also differs from what is shown in the component code section. Instead of a full code example, could it just show an example of the related `mask` attribute?